doc/doc-support/package.nix

```diff
@@ -96,15 +95,14 @@ stdenvNoCC.mkDerivation (
 
       dest="$out/share/doc/nixpkgs"
       mkdir -p "$(dirname "$dest")"
-      mv out "$dest"
-      mv "$dest/index.html" "$dest/manual.html"
+      cp -r out "$dest"
 
       cp ${roboto.src}/web/Roboto\[ital\,wdth\,wght\].ttf "$dest/Roboto.ttf"
 
       cp ${epub} "$dest/nixpkgs-manual.epub"
 
       mkdir -p $out/nix-support/
-      echo "doc manual $dest manual.html" >> $out/nix-support/hydra-build-products
+      echo "doc manual $dest index.html" >> $out/nix-support/hydra-build-products
```

Comment:
Not sure who else is may be relying that, but it's probably okay since hydra builds its own URL to that file.